schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -2668,7 +2668,7 @@ Elevator:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: False
```

Comment:
I think this should be true since it is based on occupancy and not weather, agreed?